src/ai/backend/manager/cli/__main__.py

```diff
@@ -7,13 +7,17 @@
 from pathlib import Path
 
 import click
+import psycopg2
 
 from ai.backend.common.cli import LazyGroup
 from ai.backend.common.logging import BraceStyleAdapter
+from ai.backend.common.validators import TimeDuration
 
 from ..config import load as load_config
 from ..models.keypair import generate_keypair as _gen_keypair
 from .context import CLIContext, init_logger
+from datetime import datetime
```

Comment:
Please move this import after line 2, since this is an import of the standard library module.